src/Resources/contao/classes/Frontend.php

```diff
@@ -685,4 +690,53 @@ public static function getResponseFromCache()
 
 		return null;
 	}
+
+	/**
+	 * Check authentication status based on sessionKey.
+	 *
+	 * @param string $sessionKey
+	 * @return bool
+	 *
+	 * @deprecated Deprecated since Contao 4.x, to be removed in Contao 5.0.
+	 */
+	protected function getAuthenticationStatus($sessionKey = '')
+	{
+		/** @var SessionInterface $session */
+		$session = \System::getContainer()->get('session');
+
+		if ($session->has($sessionKey) && $token = unserialize($session->get($sessionKey)))
+		{
+			/** @var TokenInterface $token */
+
+			// Validate the session ID and timeout
+			if ($token->isAuthenticated())
```

Comment:
Additional check with UserChecker

Reply:
It's now checked on request with the `TokenLifetimeListener` implemented in 4c78db7